### PR TITLE
framework/media: Find audio cards at init time

### PR DESCRIPTION
**Pull request by junmin-kim**
Call find_audio_card in audio_manager_init.
Remove init_audio_stream_out, init_audio_stream_in, init_audio_stream,
find_audio_card_if_empty.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [b79da58ea1aa01a00214059564fbc4e526a097b3] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b79da58ea1aa01a00214059564fbc4e526a097b3] - Code Rule Check (C++) OK.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -682,17 +632,26 @@ static audio_manager_result_t set_audio_volume(audio_io_direction_t direct, uint
...
+	}
+	find_audio_card(INPUT);
+
+	for (card_id = 0; card_id < CONFIG_AUDIO_MAX_OUTPUT_CARD_NUM; card_id++) {
+		pthread_mutex_init(&(g_audio_out_cards[card_id].card_mutex), NULL);
+	}
+	find_audio_card(OUTPUT);
+
+	return AUDIO_MANAGER_SUCCESS;
 }
```
Comment (juitem) [inline framework/src/media/audio/audio_manager.c]: it it okay to return AUDIO_MANAGER_SUCCESS on  find_audio_card() returns AUDIO_MANAGER_NO_VALID_CARD cases?

Comment (junmin-kim) [reply framework/src/media/audio/audio_manager.c]: Fixed it

**Comment by seinfra** [comment]
Target : [b4c1e9a642d1c847809d252ee3bd230cfadee6ce] -

**Comment by seinfra** [comment]
Target : [b4c1e9a642d1c847809d252ee3bd230cfadee6ce] -

**Comment by seinfra** [comment]
Target : [6c44763f1cc568c260385fa2a3ecaba44cecc996] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6c44763f1cc568c260385fa2a3ecaba44cecc996] - Code Rule Check OK.

**Pull request merged**